dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/keyjsonvalue/DefaultKeyJsonValueService.java

```diff
@@ -67,48 +70,81 @@ public DefaultKeyJsonValueService(
     @Transactional( readOnly = true )
     public List<String> getNamespaces()
     {
-        return keyJsonValueStore.getNamespaces();
+        List<String> namespaces = keyJsonValueStore.getNamespaces();
+        namespaces.remove( MetadataVersionService.METADATASTORE );
+
+        return namespaces;
     }
 
     @Override
     @Transactional( readOnly = true )
     public List<String> getKeysInNamespace( String namespace )
     {
+        if ( MetadataVersionService.METADATASTORE.equals( namespace ) )
```

Comment:
Instead of hard coding it like this, maybe we could have a list of "ignored" namespaces? there might be more reserved namespaces in the future..

Reply:
That sounds like a sensible idea!
On the other hand, we want to get this patch out ASAP, so we could refactor later, when we need to?

Reply:
Agree, this makes sense soon as there will be more than one "hidden" namespace.